react/features/app/actions.js

```diff
@@ -33,6 +33,10 @@ export function appNavigate(uri) {
     return (dispatch, getState) => {
         const state = getState();
         const oldDomain = getDomain(state);
+        const defaultURL = state['features/app'].app._getDefaultURL();
+        const urlObject = new URL(uri, defaultURL);
+
+        dispatch(setRoomUrl(urlObject));
```

Comment:
No it's not. You need to account for `uri` being undefined. This patch still doesn't do that.